From a31d329edfa2d73273a1a8795fd5ad602038b416 Mon Sep 17 00:00:00 2001
From: Sam Mortimer <sam@mortimer.me.uk>
Date: Wed, 22 Aug 2018 23:28:58 -0700
Subject: [PATCH] overlays/Lineage: Set LOCAL_PRIVATE_PLATFORM_APIS := true

*) Apps on pie have to set LOCAL_SDK_VERSION or
   LOCAL_PRIVATE_PLATFORM_APIS

*) Overlays are platform apps so set platform APIs

*) Unbreaks build on pie

Change-Id: I77ce4dbc53846dd617dec3dbab475253ad7689e3
---
 accents/black/Android.mk  | 2 ++
 accents/blue/Android.mk   | 2 ++
 accents/brown/Android.mk  | 2 ++
 accents/cyan/Android.mk   | 2 ++
 accents/green/Android.mk  | 2 ++
 accents/orange/Android.mk | 2 ++
 accents/pink/Android.mk   | 2 ++
 accents/purple/Android.mk | 2 ++
 accents/red/Android.mk    | 2 ++
 accents/yellow/Android.mk | 2 ++
 black/Android.mk          | 2 ++
 dark/Android.mk           | 2 ++
 12 files changed, 24 insertions(+)

diff --git a/accents/black/Android.mk b/accents/black/Android.mk
index 5073709..ed759ab 100644
--- a/accents/black/Android.mk
+++ b/accents/black/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineageBlackAccent
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/accents/blue/Android.mk b/accents/blue/Android.mk
index 74a6c4f..b3c2264 100644
--- a/accents/blue/Android.mk
+++ b/accents/blue/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineageBlueAccent
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/accents/brown/Android.mk b/accents/brown/Android.mk
index 744f61d..67c19d3 100644
--- a/accents/brown/Android.mk
+++ b/accents/brown/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineageBrownAccent
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/accents/cyan/Android.mk b/accents/cyan/Android.mk
index 6538ef1..b5d040b 100644
--- a/accents/cyan/Android.mk
+++ b/accents/cyan/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineageCyanAccent
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/accents/green/Android.mk b/accents/green/Android.mk
index e6279de..63084c0 100644
--- a/accents/green/Android.mk
+++ b/accents/green/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineageGreenAccent
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/accents/orange/Android.mk b/accents/orange/Android.mk
index ea95cf8..dfedd38 100644
--- a/accents/orange/Android.mk
+++ b/accents/orange/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineageOrangeAccent
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/accents/pink/Android.mk b/accents/pink/Android.mk
index 72282fd..0b8e428 100644
--- a/accents/pink/Android.mk
+++ b/accents/pink/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineagePinkAccent
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/accents/purple/Android.mk b/accents/purple/Android.mk
index 47a6d0f..0a3677b 100644
--- a/accents/purple/Android.mk
+++ b/accents/purple/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineagePurpleAccent
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/accents/red/Android.mk b/accents/red/Android.mk
index fa54502..d297312 100644
--- a/accents/red/Android.mk
+++ b/accents/red/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineageRedAccent
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/accents/yellow/Android.mk b/accents/yellow/Android.mk
index 9481bd7..9a1c646 100644
--- a/accents/yellow/Android.mk
+++ b/accents/yellow/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineageYellowAccent
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/black/Android.mk b/black/Android.mk
index ff2e000..a346fa6 100644
--- a/black/Android.mk
+++ b/black/Android.mk
@@ -6,4 +6,6 @@ LOCAL_CERTIFICATE := platform
 LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineageBlackTheme
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
diff --git a/dark/Android.mk b/dark/Android.mk
index dacc24d..feb39db 100644
--- a/dark/Android.mk
+++ b/dark/Android.mk
@@ -7,4 +7,6 @@ LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
 LOCAL_PACKAGE_NAME := LineageDarkTheme
 LOCAL_OVERRIDES_PACKAGES := SysuiDarkThemeOverlay
 
+LOCAL_PRIVATE_PLATFORM_APIS := true
+
 include $(BUILD_RRO_SYSTEM_PACKAGE)
-- 
2.17.1

